engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -218,6 +245,10 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
     s"$workflowId-BackendJobExecutionActor-${jobDescriptor.key.tag}"
   }
 
+  private def buildCacheHitCopyingActorName(jobDescriptor: BackendJobDescriptor) = {
+    s"$workflowId-BackendCacheHitCopyingActor-${jobDescriptor.key.tag}"
```

Comment:
I've already removed at least one situation like this. Much like our "30,000 copies of a stack trace" pattern, this is pretty common as well